view/settings/index.ts

```diff
@@ -1,17 +1,23 @@
 import { WritableAtom } from 'nanostores'
 
-import { showCharts, showP3, showRec2020 } from '../../stores/settings.js'
+import {
+  showCharts,
+  showP3,
+  showRec2020,
+  showModel
+} from '../../stores/settings.js'
 import { getCheckbox, onChange } from '../checkbox/index.js'
 
 function init(store: WritableAtom<boolean>, checkbox: HTMLInputElement): void {
   store.subscribe(show => {
     checkbox.checked = show
   })
-  onChange(checkbox, checked => {
+  onChange(checkbox, async checked => {
```

Comment:
Why do we need it?

Reply:
Accidentally left it in from earlier version, removed now